Procfile

```diff
@@ -1 +1,2 @@
+release: POOL_SIZE=2 MIX_ENV=$MIX_ENV mix ecto.migrate
```

Comment:
hey @keathley - curious why POOL_SIZE must be 2?

Reply:
Heroku's max connections for PG instances is 20. 18 of those are allocated for web workers. So we have to specify 2 here or else we exceed the 20 connection limit.

Reply:
Ah, gotcha! ✌️